astropy/table/table.py

```diff
@@ -2372,7 +2374,8 @@ def _replace_column_warnings(self, name, col):
             except AttributeError:
                 pass
 
-        if 'refcount' in warns:
+        # sys.getrefcount is CPython specific and not on PyPy.
+        if 'refcount' in warns and hasattr(sys, 'getrefcount'):
```

Comment:
Can `refcount` be in `warns` and we do not have `getrefcount` available? (I.e., did this actually fail on PyPy?)

Reply:
It's one of the options it seems:
https://github.com/astropy/astropy/blob/6400c89e58add0e8b96e00f26dcd48b877c2acc9/astropy/table/__init__.py#L33-L38

Reply:
OK, I should have checked...